content/docs/guides/contribute/contribute-to-code/_index.en.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
the name of the folder is odd: `contribute-to-code`should be `contribute-code`

Reply:
Done!